TX/Generic LR1121 Gemini.json

```diff
@@ -0,0 +1,40 @@
+{
+    "serial_rx": 1,
+    "serial_tx": 1,
```

Comment:
For a generic target I don't think we want to be using the main serial port for this do we?

Reply:
Fixed to match dev board.